sdks/java/core/src/test/java/org/apache/beam/sdk/coders/CoderRegistryTest.java

```diff
@@ -89,8 +90,14 @@ public void testSerializableFallbackCoderProvider() throws Exception {
   public void testProtoCoderFallbackCoderProvider() throws Exception {
     CoderRegistry registry = getStandardRegistry();
 
+    //must use reflection here or a reference to dataflow class is baked into
```

Comment:
The problem is that these unit tests are being run in the direct-runner `integration-test` target; they shouldn't because of the way we filter for `@Category(NeedsRunner.class)` using the `<groups/>` config.

I don't like the proposed solution. We are running redundant tests, and we'll run into this same failure in the future if we add other test dependencies.

I dug into this a bit more, and can verify that these extra tests are running when using `mvn -Peclipse-jdt clean integration-test` but not `mvn clean integration-test`. Searching around, [this thread](http://stackoverflow.com/a/36718377/33791) asserts that Tycho supports `<groups/>` configuration, but not until version 0.25. Can you try bumping the dependency version and see if that fixes it?